src/Runner/Hook.php

```diff
@@ -194,6 +211,16 @@ public function shouldSkipActions(?bool $shouldSkip = null): bool
         return $this->skipActions;
     }
 
+    /**
+     * Returns all actions configured for this hook or applicable virtual hooks
+     *
+     * @return \CaptainHook\App\Config\Action[]
+     */
+    public function getActions(): array
+    {
+        return $this->getActionsToExecute($this->getHookConfigsToHandle());
+    }
```

Comment:
This allows the hook command class access to the list of actions configured for the hook, so it may use these to list the actions, when `--list-actions` is provided.